utils/kubernetes/service.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package kubernetes
+
+import (
+	"net/url"
+
+	v1 "k8s.io/api/core/v1"
+	"knative.dev/pkg/apis"
+)
+
+// TODO: retrieve the cluster domain from the /etc/resolve inside the pod or from the Platform CRD
+var defaultClusterDomain = "svc.cluster.local"
```

Comment:
Can you open a JIRA for this TODO?

Reply:
Created [KOGITO-9198](https://issues.redhat.com/browse/KOGITO-9198)